site/Makefile

```diff
@@ -131,3 +133,9 @@ release-notes:
 # To override: make execute PROFILE=select-profile FILE_PATH=notebooks/notebook-path-here.ipynb
 execute:
 	quarto render --profile $(PROFILE) $(FILE_PATH)
+
+docker-image: docs-site
+	@echo "\nBuilding the Docker image ..."
+	@docker build -f ../Dockerfile -t validmind-docs ..
+	@echo "\nStarting the Docker container on http://localhost:4444/ ..."
+	@docker run -p 4444:80 validmind-docs && echo "Access the docs site at: http://localhost:4444/"
```

Comment:
Maybe worth having separate "build" and "run" docker steps but definitely not required.

Reply:
Good idea — I split the original action into `docker-build` and `docker-serve` via https://github.com/validmind/documentation/pull/594/commits/be498e3a885c6feab51e371b034e9b7bec6fc6fe and updated the readme. Re-tested, works as expected.